cloudevents/extensions/deprecation.md

```diff
@@ -0,0 +1,86 @@
+# Deprecation extension
+
+This specification defines attributes that can be included in CloudEvents to
+indicate to [consumers](../spec.md#consumer) or
+[intermediaries](../spec.md#intermediary) the deprecation of the events. These
+attributes inform consumers of CloudEvents about upcoming changes or removals,
+facilitating smoother transitions and proactive adjustments.
+
+## Notational Conventions
+
+As with the main [CloudEvents specification](../spec.md), the key words "MUST",
+"MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT",
+"RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as
+described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+However, the scope of these key words is limited to when this extension is
+used. For example, an attribute being marked as "REQUIRED" does not mean it
+needs to be in all CloudEvents, rather it needs to be included only when this
+extension is being used.
+
+## Attributes
+
+### deprecation
+
+- Type: `Boolean`
+- Description: Indicates whether the event type is deprecated.
+- Constraints
+  - REQUIRED
+  - The value MUST be a boolean (`true` or `false`).
+- Example: `"deprecation": "true"`
```

Comment:
I'm torn between the consistency of having a common prefix, and the oddity of it sounding like it's not Boolean (whereas "deprecated: true" sounds fine).

Thinking about it more, when would this ever actually have a value of false? That would be unusual at least...

Reply:
Does this have to be REQUIRED? 

If you just have a `deprecated`: true and imply a default of false, there is no need to add a mandatory attribute.

Reply:
renamed to deprecated